src/GraphQL/Buffers/EntityBuffer.php

```diff
@@ -67,19 +71,22 @@ public function resolveBufferArray(array $buffer) {
       ->getStorage($type)
       ->loadMultiple($ids);
 
-    return array_map(function ($item) use ($entities) {
+    return array_map(function ($item) use ($entities, $language) {
       if (is_array($item['id'])) {
-        return array_reduce($item['id'], function ($carry, $current) use ($entities) {
+        return array_reduce($item['id'], function ($carry, $current) use ($entities, $language) {
           if (!empty($entities[$current])) {
-            array_push($carry, $entities[$current]);
+            $entity = $language ? $entities[$current]->getTranslation($language) : $entities[$current];
```

Comment:
this will throw fatal errors if the entity type is not translatable. That is also why PHPStan is complaining. Maybe that is ok and we don't care? We could already throw an exception if a language is passed in the add() method on not translatable entity types.